From 6758574e917cde2a9efd42f6cb1312f9819f04ff Mon Sep 17 00:00:00 2001
From: csakoda <cms235@gmail.com>
Date: Thu, 22 Oct 2015 14:22:27 -0700
Subject: [PATCH] Fix kombu and django-celery versions

Kombu 3.0.26 and django-celery 3.1.16 are the last verisons that work with django 1.6 it seems
---
 requirements-base.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/requirements-base.txt b/requirements-base.txt
index c95e859..93ba4f4 100644
--- a/requirements-base.txt
+++ b/requirements-base.txt
@@ -22,7 +22,8 @@ db_backup==0.1.3
 boto==2.26.1
 lxml==3.3.3
 celery>=3.1,<3.2
-django-celery>=3.1,<3.2
+django-celery>=3.1,<3.1.17
 importlib
 django-social-auth>=0.7.9
 paramiko==1.15.2
+kombu==3.0.26

